cypress/utils/table.js

```diff
@@ -89,6 +89,8 @@ function checkFiltering(
     checkTableHeaders(tableHeaders);
   } else {
     cy.get('[data-ouia-component-id=loading-skeleton]').should('not.exist');
+    // eslint-disable cypress/no-unnecessary-waiting
+    cy.wait(1000);
```

Comment:
I think this is a very good example of where we actually don't need unnecessary waiting :-)

Can we think of why our tests are timing out or not able to find the expected data under the columns? Is it the fact that the table is still in the loading state? If so, what should appear if we consider the table loaded? Instead of waiting generic 1 second, can we just wait for some of the first row data to appear?

Reply:
Yeah I wanted to give it a try if it eventually helps something, looks like it doesn't

Reply:
Loading state I've fixed few lines below by adding this
`cy.get('[data-ouia-component-id=loading-skeleton]').should('not.exist');`
Waits until the skeleton is gone